packages/wrangler/src/__tests__/versions/versions.deploy.test.ts

```diff
@@ -925,39 +924,28 @@ describe("versions deploy", () => {
 
 describe("units", () => {
 	describe("parseVersionSpecs", () => {
-		const options = yargs().command(
-			"versions deploy [version-specs..]",
-			"",
-			// @ts-expect-error creating the command using a fresh yargs() but it expects one preconfigured with global options
-			versionsDeployOptions,
-			() => {}
-		);
-
 		test("no args", () => {
-			const input = "versions deploy";
-
-			const args = options.parse(input) as VersionsDeployArgs;
-			const result = parseVersionSpecs(args);
+			const result = parseVersionSpecs({});
```

Comment:
These are now pure unit tests, rather than testing any yargs functionality. So the ordering tests have been removed (not the concern of this function), and there is no need to build and parse the yargs object.